### DISCUSSION TITLE
Project Folder Icon Color

### DISCUSSION THREAD

**Discussion by CrimsnShadow** [FiercestT/EasyEdaThemes#6] [Q&A] [answered]
How do I change the color of the Project Folder Icon?

_Image Attached_
![project-folder](https://user-images.githubusercontent.com/29812341/143718907-e4066ed8-4a96-40c3-be29-b2878c4552c6.PNG)

**Comment by FiercestT** [comment, marked as answer]
You can't at the moment. I just checked and it's an easy addition. I will add it and base it off of the primary color of the selected theme.

Thanks for letting me know about that 👍

**Reply by CrimsnShadow** [reply]
Awesome! Thanks :)

**Reply by FiercestT** [reply]
Introduced in v1.2.1.

https://github.com/FiercestT/EasyEdaThemes/releases/tag/v1.2.1
